label_studio_ml/examples/segment_anything_model/labelstudio.py

```diff
@@ -0,0 +1,205 @@
+import torch
+from torchvision import transforms
+import matplotlib.pyplot as plt
+from segment_anything import sam_model_registry, SamPredictor
+from segment_anything.utils.onnx import SamOnnxModel
+from label_studio_ml.model import LabelStudioMLBase
+from label_studio_converter import brush
+from label_studio_ml.utils import get_image_local_path
+import numpy as np
+import cv2
+import os
+from PIL import Image
+import string
+import random
+
+import onnxruntime
+from onnxruntime.quantization import QuantType
+from onnxruntime.quantization.quantize import quantize_dynamic
+
+def load_my_model():
+        """
+        Loads the Segment Anything model on initializing Label studio, so if you call it outside MyModel it doesn't load every time you try to make a prediction
+        Returns the predictor object. For more, look at Facebook's SAM docs
+        """
+        device = "cuda"     # if you're not using CUDA, use "cpu" instead .... good luck not burning your computer lol
+        sam = sam_model_registry["vit_h"](checkpoint="sam_vit_h_4b8939.pth")        # Note: YOU MUST HAVE THE MODEL SAVED IN THE SAME DIRECTORY AS YOUR BACKEND
```

Comment:
@shondle Could you please extract checkpoint path to an environment variable or model param?
The file name would change in some point of time.

Reply:
Done.